spec/catalog.yaml

```diff
@@ -261,6 +286,22 @@ components:
           default: { }
           nullable: true
 
+    PageToken:
+      description:
+        An opaque token that allows pagination for list APIs (e.g. ListNamespaces).
+        
+        For an initial client request for a list API, if the server cannot return all items in one response,
```

Comment:
note that this is a much simplified version of the one in IRC, because in IRC we did not have pagination for a long time until we realize production users typically could have many namespaces and tables. And we had to add complicated rules for backwards compatibility. Since we are starting from fresh here, it does not need to have those complications.